docs/Erlang Distribution.md

```diff
@@ -0,0 +1,92 @@
+# Erlang Distribution
+
+## Static Configuration
+
+The simplest way to configure Erlang distribution is to statically (i.e., at
+compile time) set the node name and cookie. This is done by editing the
+`rel/vm.args` file, which contains the command-line arguments used to launch
+the Erlang/OTP virtual machine.
+
+You would simply add `-name` and `-setcookie` parameters, as follows:
```

Comment:
I had understood that `-sname` is only for nodes on the same host. Had I misunderstood that?

As for `.local`, there isn't anything special about the extension as such. You can name your local hosts with that extension with or without mDNS. We need some example name to use in the docs, so I used that.